### ISSUE TITLE
Feature Request: Gsudo program icon to display in UAC?

### ISSUE THREAD

**Issue by e-t-l** [gerardog/gsudo#371]
## Description

I know this is a very low-priority request. It is kind of annoying/ugly to see the Windows generic program icon in the UAC prompt every time gsudo is run. 

## Proposed technical details

An icon just needs to be designed (or found, with the appropriate free usage license) and packaged as the gsudo.exe's icon resource. I have limited graphic design skills and could try making something; some other gsudo user might be able to do better. You may have some preferences or design ideas about what such an icon might look like (or you might prefer not to have one at all!)

Do you think you would you be interested in having an icon/logo to represent gsudo?

**Comment by gerardog** [comment]
Hey! An icon is welcomed! I dont have graphic skills. Currently the web ( https://gerardog.github.io/gsudo )
shows a terrible red sharp as web icon: https://gerardog.github.io/gsudo/img/favicon.ico 
Maybe we can put the red sharp in front of a shield?  Or something else.
Feel free to paste ideas!
Thanks

**Comment by LukeMitsialis** [comment]
Hello! I mocked something up real quick, is this close to what your looking for?
![image](https://github.com/user-attachments/assets/f1d27cf5-3504-4a81-a619-1118bd66dff6)
Heres the image by itself
![image](https://github.com/user-attachments/assets/78c20c16-20f0-4e20-a50d-7a353ba38eb8)

Cheers,

**Comment by LukeMitsialis** [comment]
I also like this one
![image](https://github.com/user-attachments/assets/0952c695-8ec8-4c76-8ebc-75e69cc64e43)

logo by itself:
![image](https://github.com/user-attachments/assets/8b5d42ce-a620-416f-9a84-356684d1d461)

**Comment by LukeMitsialis** [comment]
<img width="695" alt="image" src="https://github.com/user-attachments/assets/9882557a-2e93-4314-9936-9fdfeef2f9e4">

I settled on this iteration. its got got legibility at scale and i like the dark background with the red sharp hinting towards how it can look in the terminal.

**Comment by e-t-l** [comment]
Personally, I think it looks great. What does gerardog think? Have you opened a PR for this?

**Comment by gerardog** [comment]
Hey, i love it, but i think that the outer border may look cooler in white or a more calm tone!
Share the source file so I can add it, or submit a PR. Thanks!

**Comment by gerardog** [comment]
![image](https://github.com/user-attachments/assets/fb8f20e7-d907-4ee7-887c-7a877d07a712)

**Comment by LukeMitsialis** [comment]
![gsudo-logoidea](https://github.com/user-attachments/assets/2e76c393-cb17-4abc-8e55-47334eccaf96)
![gsudo-logoidea2](https://github.com/user-attachments/assets/6cc17aba-b73b-4962-9997-e4b11b4a0847)


Not sure how to make a pr as this is my first contribution, I honestly prefer the red option provided as it has more personality and color harmony, but ive included the grey option if thats your preference. Ill learn how to submit PRs for next time 😅

**Comment by gerardog** [comment]
The bottom left one is a real sudo build, the others are just rough edited.

Win 11 default theme:
![image](https://github.com/user-attachments/assets/eadecf65-bcee-4c0e-8344-e929cd3956f6)

Win 10 dark:
![image](https://github.com/user-attachments/assets/e6af2d88-1083-485a-b216-a29031bfab11)

React with a 🎉 for grey one
React with a 🚀 for red one
Or add a comment.
I like both.

**Comment by e-t-l** [comment]
I agree the gray is a more calm tone, but I actually think that having a brighter, more alarming red color is appropriate for a powerful program like Gsudo. To me, it emphasizes: "HEY! This program can majorly screw up your pc! Are you SURE you want to allow it?"

(small detail, but I personally think it looks cleaner _without_ the medium-dark red accent stroke around the sharp symbol.)

**Comment by gerardog** [comment]
Ok, this is shipping in the next release. Thanks for your patience. 

If you want to propose alternate icons, I suggest you start with [gsudo.svg](/master/src/gsudo/icon/gsudo.svg)
I've used `icofx 3 trial edition` to convert from `svg` to `ico`.

I would accept PR's with a better icon. I've never been good at image edition, please include an updated `.svg` file too.

![Image](https://github.com/user-attachments/assets/c65b38a1-0780-4994-ae7d-4c47023757a9)
